app/config/locale/translations/id.countries.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 return [
-    'AF' => 'Afganistan',
+    'AF' => 'Afghanistan',
```

Comment:
According to my understanding, Afghanistan  => Afganistan   refer to [this ](https://www.101languages.net/indonesian/country-names-indonesian/)

Reply:
i misread the title on wikipedia sorry.